srcpkgs/pony-stable/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'pony-stable'
+pkgname=pony-stable
+version=0.1.4
+revision=1
+only_for_archs="x86_64"
+hostmakedepends="ponyc"
+makedepends="libatomic-devel"
+short_desc="A simple dependency manager for the Pony language"
+maintainer="Brian Mitchell <brian@strmpnk.co>"
+license="2-clause-BSD"
```

Comment:
Use the spdx-identifier `BSD-2-Clause`

Reply:
Thanks. I had copied it from the earlier packages. I'll change both.